src/extension/byok/node/azureOpenAIEndpoint.ts

```
@@ -0,0 +1,147 @@
+		return this.isOSeriesModel(modelId);
+	}
+
+	override interceptBody(body: IEndpointBody | undefined): void {
+		super.interceptBody(body);
+
+		if (body && this.isOSeriesModel(this.model)) {
+			// O-series models in Azure OpenAI only support temperature = 1 (default)
+			// Remove temperature parameter to avoid the "Unsupported value: 'temperature' does not support 0.1" error
+			if ('temperature' in body) {
+				console.info(`Azure o-series model ${this.model}: Removing temperature parameter (${body.temperature}) - o-series models only support default temperature=1`);
+				delete body.temperature;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)